flutter/example/lib/main.dart

```diff
@@ -199,6 +210,13 @@ class CocoaExample extends StatelessWidget {
             await channel.invokeMethod<void>('crash');
           },
         ),
+        RaisedButton(
+          child: const Text('Crash swift'),
+          onPressed: () async {
+            const channel = MethodChannel('example.flutter.sentry.io');
+            await channel.invokeMethod<void>('crashSwift');
+          },
+        ),
```

Comment:
this is not part of this PR right? we don't have this method on the flutter plugin

Reply:
✅ ooops, thanks